deploy/mainnet/481_deploy_frax3pool.ts

```diff
@@ -0,0 +1,78 @@
+import { HardhatRuntimeEnvironment } from "hardhat/types"
+import { DeployFunction } from "hardhat-deploy/types"
+
+// Deployment Names
+const BASE_POOL_NAME = "SaddleFrax3Pool"
+const BASE_POOL_LP_TOKEN_NAME = `${BASE_POOL_NAME}LPToken`
+
+// Constructor arguments
+const TOKEN_NAMES = ["USDC", "USDT", "FRAX"]
+const TOKEN_DECIMALS = [6, 6, 18]
+const LP_TOKEN_NAME = "Saddle USDC/USDT/FRAX LP Token"
+const LP_TOKEN_SYMBOL = "saddleUSDCUSDTFrax"
+const INITIAL_A = 500
+const SWAP_FEE = 4e6 // 4bps
+const ADMIN_FEE = 5e9 // 50% of the 3bps
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { deployments, getNamedAccounts } = hre
+  const { execute, get, getOrNull, log, read, save } = deployments
+  const { deployer } = await getNamedAccounts()
+
+  // Manually check if the pool is already deployed
+  const saddle4pool = await getOrNull(BASE_POOL_NAME)
```

Comment:
`saddle4pool` should be renamed to something more generic here.

Reply:
renamed to poolContract